From 9a55066301f29fb94e532b099ef089786f198ba7 Mon Sep 17 00:00:00 2001
From: huzhen <george.hu@spacemit.com>
Date: Tue, 25 Jun 2024 20:24:19 +0800
Subject: [PATCH 0770/1206] qspi: fix the warning when disable the clk and bus
 clk of qspi

  As the clk and bus clk of qsip is already disabled when initialed,
  so it is not need to disabled.

Change-Id: I2d2b6fde9a43426dd398a2e8d061f7013c53748a
---
 drivers/spi/spi-k1x-qspi.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/spi/spi-k1x-qspi.c b/drivers/spi/spi-k1x-qspi.c
index c5a6d46597c0..88a73be7d29e 100644
--- a/drivers/spi/spi-k1x-qspi.c
+++ b/drivers/spi/spi-k1x-qspi.c
@@ -403,9 +403,6 @@ static int qspi_set_func_clk(struct k1x_qspi *qspi)
 		return -EINVAL;
 	}
 
-	clk_disable_unprepare(qspi->bus_clk);
-	clk_disable_unprepare(qspi->clk);
-
 	ret = clk_set_rate(qspi->clk, qspi->max_hz);
 	if (ret) {
 		dev_err(qspi->dev, "fail to set clk, ret:%d\n", ret);
-- 
2.47.0

